packages/@lwc/integration-karma/test/template/html-comments/x/confusedWithText/confusedWithText.html

```diff
@@ -0,0 +1,8 @@
+<template lwc:preserve-comments>
```

Comment:
Does this issue exist if we don't have `lwc:preserve-comments`?

Reply:
no, because for comments to be rendered, the root template tag needs to have the `lwc:preserve-comments` directive.